wordpress-mobile/gutenberg-mobile
# Add upload test cases for VideoPress block

**PRs related:**
* https://github.com/WordPress/gutenberg/pull/50398
* https://github.com/WordPress/gutenberg/pull/50399
* https://github.com/Automattic/jetpack/pull/30486

Part of https://github.com/wordpress-mobile/gutenberg-mobile/issues/5734.

This PR adds several test cases to cover the upload functionality of the VideoPress. block.

**To test:**
1. Run the command: `TEST_RN_PLATFORM=ios npm run test`
2. Run the command: `TEST_RN_PLATFORM=android npm run test`
3. Check that all CI jobs succeed.

PR submission checklist:
